src/js/resolver.js

```diff
@@ -14,9 +14,28 @@ Resolver.prototype.resolve = function(spec, callback, scope) {
   // models
   for(name in spec.definitions) {
     var model = spec.definitions[name];
-    for(propertyName in model.properties) {
-      property = model.properties[propertyName];
-      this.resolveTo(property, resolutionTable);
+    if (!model.properties) {
+      model.properties = {};
+    }
+    if (model.allOf) {
+      for(mergeName in model.allOf){
```

Comment:
this would introduce a dependency on underscorejs - it is not yet a requirement for swagger-js and I'm not sure if I should really add it